submissions/description/server_security_misconfiguration/potentially_unsafe_http_method_enabled/options/guidance.md

```diff
@@ -0,0 +1,9 @@
+# Guidance
+
+<!--
+Provide a 1-2 sentence description - see http://cveproject.github.io/docs/content/key-details-phrasing.pdf for tips.
+Also provide a step-by-step walkthrough with screenshot on how you exploited the vulnerability. This will speed triage time and result in faster rewards!
+
+Your submission MUST include evidence of the vulnerability and not be theoretical in nature.
+Attempt to escalate the vulnerability to perform additional actions. If this is possible, provide a full proof-of-concept.
+-->
```

Comment:
HTML comment to be removed

Reply:
Thank you, I missed these tags. Will remove now